app/controllers/api/v0/swagger/models/highlight.rb

```diff
@@ -42,25 +42,69 @@ module Api::V0::Swagger::Models::Highlight
     end
   end
 
+  swagger_schema :GetHighlights do
+    key :required, [:source_type]
+    property :page do
+      key :type, :integer
+      key :description, 'The page number'
+    end
+    property :per_page do
+      key :type, :integer
+      key :description, 'The number of highlights to return per page'
+    end
+    property :order do
+      key :type, :string
+      key :enum, %w[asc desc]
+      key :description, 'The returned sorted order'
+    end
+  end
+
   swagger_schema :Highlight do
     key :required, [:id]
   end
 
+  swagger_schema :Highlights do
+    property :highlights do
+      key :type, :array
+      key :description, 'One or more highlights'
+      items do
+        key :'$ref', :Highlight
+      end
+    end
+  end
+
   swagger_schema :NewHighlight do
     key :required, [:source_type, :source_id, :anchor, :highlighted_content, :color, :location_strategies]
   end
 
+  add_properties(:GetHighlights, :NewHighlight, :Highlight) do
+    property :source_type do
+      key :type, :string
+      key :enum, ['openstax_page']
+      key :description, 'The source_type of the highlight, typically a openstax_page'
+    end
+    property :source_parent_ids do
+      key :type, :array
+      key :description, 'The parent IDs of the highlight. For book highlights, ' \
+                        'the parent_id could be a book, unit, or chapter ID'
```

Comment:
"parent IDs of the highlight" isn't quite right.  sounds like parents of the highlights when really it is parents of the source the highlight is in.  "IDs of the parent sources of the source containing the highlight.  E.g. For an openstax_page highlight, the IDs would be those of the chapter, unit, and book.

Also note that it should be "and" instead of "or"

Does the order of these IDs matter in the array?  If not say that.